datarepo-clienttests/src/main/resources/configs/functional/BillingProfileAccess.json

```diff
@@ -17,5 +17,5 @@
       "expectedTimeForEachUnit": "HOURS"
     }
   ],
-  "testUserFiles": ["dumbledore.json", "voldemort.json", "mcgonagall.json"]
+  "testUserFiles": ["dumbledore.json", "harry.json", "mcgonagall.json"]
```

Comment:
It's less of an issue now but there used to be a line that would require removing the stewards group since voldemort is in there and would inherit all of those permissions.  To future proof, I wanted to use a user that isn't in the admins group either